src/app/example/petForm.tsx

```diff
@@ -0,0 +1,196 @@
+    defaultValues: {
+      "dog-name": "",
+      "dog-preferred-vets": [],
+      "dog-weight": "",
+    },
+  });
+
+  async function onSubmit(values: FormDataType) {
+    // ! Send the form data to the server.
+    console.log(values);
+    onSubmitForm(values);
+  }
+
+  return (
+    <>
+      <BarkForm onSubmit={onSubmit} form={form}>
+        <BarkFormHeader>Tell us about your pet</BarkFormHeader>
+        <BarkFormInput
+          form={form}
+          label="What’s your dog’s name?"
+          name="dog-name"
+        />
+
+        <BarkFormSelect
+          form={form}
+          label="What’s your dog’s breed?"
+          name="dog-breed"
+          options={breeds.map((breed) => ({
+            label: breed.dog_breed,
+            value: breed.dog_breed,
+          }))}
+        />
+
+        <BarkFormDatetimeInput
+          form={form}
+          label="When is it’s birthday? (DD/MM/YYYY)"
+          name="dog-birthday"
+        />
+
+        <BarkFormRadioGroup
+          form={form}
+          label="What’s your dog’s sex?"
+          name="dog-sex"
+          layout="button"
+          options={[
+            { label: "Yes", value: "yes" },
+            {
+              label: "No",
+              value: "no",
+            },
+          ]}
+        />
+
+        <BarkFormInput
+          form={form}
+          label="What’s your dog’s weight?"
+          name="dog-weight"
+          type="number"
+        />
+
+        <BarkFormRadioGroup
+          form={form}
+          label="Do you know it’s blood type?"
+          name="dog-blood-type"
+          options={[
+            { label: "I don't know", value: "idk" },
+            {
+              label: "D.E.A 1.1 positive",
+              value: "dea1.1-positive",
+            },
+            {
+              label: "D.E.A 1.1 negative",
+              value: "dea1.1-negative",
+            },
+          ]}
+        />
+
+        <BarkFormRadioGroup
+          form={form}
+          label="Has it received blood transfusion before?"
+          name="dog-blood-transfusion-status"
+          layout="button"
+          options={[
+            { label: "I don't know", value: "idk" },
+            {
+              label: "Yes",
+              value: "yes",
+            },
+            {
+              label: "No",
+              value: "no",
+            },
+          ]}
+        />
+
+        <BarkFormRadioGroup
+          form={form}
+          label="Has your dog been pregnant before?"
+          name="dog-pregnant-status"
+          layout="button"
+          options={[
+            { label: "I don't know", value: "idk" },
+            {
+              label: "Yes",
+              value: "yes",
+            },
+            {
+              label: "No",
+              value: "no",
+            },
+          ]}
+        />
+
+        <BarkFormDatetimeInput
+          form={form}
+          label="When was it’s last blood donation? (DD/MM/YYYY)"
```

Comment:
The label is incorrect

Reply:
ok but input for heartworm vaccination and last donation seem to be removed from figma so I will remove these fields as well.